activerecord/lib/active_record/associations/association.rb

```diff
@@ -40,7 +40,9 @@ def reset
       end
 
       # Reloads the \target and returns +self+ on success.
-      def reload
+      # The QueryCache is cleared if +force+ is true.
```

Comment:
Should we say `truthy` instead of `true` here?

Reply:
Thanks, @kamipo.
There are some places in the code where I see "truthy" being used so I asked.

Eg: https://github.com/rails/rails/blob/99c87ad2474d5c5b6e52ceac34c3cf9f9cb57f9f/activerecord/lib/active_record/nested_attributes.rb#L444